src/oscar/apps/dashboard/ranges/views.py

```diff
@@ -190,12 +250,17 @@ def handle_file_products(self, request, range, form):
             messages.success(request, msg, extra_tags='safe noicon block')
         self.check_imported_products_sku_duplicates(request, products)
 
-    def create_upload_object(self, request, range, f):
+    def create_upload_object(self, request, range, f, included=True):
```

Comment:
Same as above, you can remove included here, because you have the upload type in `f.cleaned_data["upload_type"]`